package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "license": "GPL-3.0-or-later",
+  "private": true,
+  "browserslist": [
+    "> 0.5%",
+    "last 2 versions",
+    "Firefox ESR",
+    "not ie <= 10",
+    "not ie_mob <= 10",
+    "not bb <= 10",
+    "not op_mob <= 12.1"
+  ],
+  "scripts": {
+    "start": "webpack --watch",
+    "build": "webpack --mode production",
+    "pipeline": "yarn install;yarn build"
+  },
+  "dependencies": {
+    "compass-mixins": "^0.12.10"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/plugin-proposal-object-rest-spread": "^7.6.2",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.6.3",
+    "@babel/preset-typescript": "^7.6.0",
+    "@types/mini-css-extract-plugin": "^0.8.0",
+    "@types/terser-webpack-plugin": "^1.2.1",
+    "@types/webpack": "^4.39.3",
+    "autoprefixer": "^9.6.5",
+    "babel-loader": "^8.0.4",
+    "cache-loader": "^4.1.0",
+    "css-loader": "^3.2.0",
+    "css-mqpacker": "^7.0.0",
+    "cssnano": "^4.1.10",
+    "mini-css-extract-plugin": "^0.8.0",
+    "node-sass": "^4.11.0",
+    "node-sass-glob-importer": "^5.3.2",
+    "pleeease-filters": "^4.0.0",
+    "postcss-assets": "^5.0.0",
+    "postcss-calc": "^7.0.1",
+    "postcss-clip-path-polyfill": "^1.0.4",
+    "postcss-easing-gradients": "^3.0.1",
+    "postcss-easings": "^2.0.0",
+    "postcss-flexbugs-fixes": "^4.1.0",
+    "postcss-focus": "^4.0.0",
+    "postcss-loader": "^3.0.0",
+    "postcss-momentum-scrolling": "^1.9.15",
+    "postcss-normalize": "^8.0.1",
+    "postcss-preset-env": "^6.7.0",
+    "postcss-pxtorem": "^4.0.1",
+    "postcss-quantity-queries": "^0.5.0",
+    "postcss-reporter": "^6.0.1",
+    "postcss-responsive-type": "^1.0.0",
+    "postcss-round-subpixels": "^1.2.0",
+    "postcss-url": "^8.0.0",
+    "postcss-vmax": "^1.0.0",
+    "prettier": "^1.18.2",
+    "prettier-loader": "^3.2.0",
+    "sass-loader": "^8.0.0",
+    "sort-css-media-queries": "^1.4.3",
+    "terser-webpack-plugin": "^2.1.3",
+    "ts-loader": "^6.2.0",
```

Comment:
If we don't use typescript and other things in this package, we also shouldn't have all those dependencies

Reply:
I just want it to have the same as Neos.io. Currently it is used as the webpack config

Reply:
I have removed TypeScript and all React